Static/DataSource.swift

```diff
@@ -188,11 +187,11 @@ extension DataSource: UITableViewDataSource {
     }
 
     public func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
     }
     
     public func tableView(tableView: UITableView, estimatedHeightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return rowForIndexPath(indexPath)?.height ?? 44
+        return rowForIndexPath(indexPath)?.height ?? UITableViewAutomaticDimension
```

Comment:
I believe this API exists to ameliorate performance problems due to excessive layout calculations when the content of any given row (and size requirements of the cell to accommodate that content) cannot be known up front. A table view that shows data that came over the network, for instance.

Since the whole point of Static is building a table view with content that _is_ known at compile time, I'm struggling to think of a case where providing an estimated height can justify adding yet another property to the API.

As it stands now, if you want a cell to have a fixed height regardless of content, you specify that in the `Row` initializer and then that value is also used for the estimation which is perfect because that's how tall the row will be. Alternatively, if you're using AutoLayout without an explicit height, any estimation only helps the table view defer geometry calculations from load time to scroll time. In that case, is providing a bad estimation better than providing no estimation at all? Should we return 44 only if the provided height on the row wasn't explicitly set?